### ISSUE TITLE
CSGO Crash and Vanguard

### ISSUE THREAD

**Issue by TauCSGO** [ThaisenPM/Loader2#46]
I have tried the injector and I like it, it is simple and useful
But after injecting, the csgo crashed, I assume it must not be updated
I am a Vanguard loaders customer, and I am looking for an alternative because Vanguard is going to close on February 1
They say in discord that on February 1 they will give an alternative, but the clients of my small community cannot be without a loader for a few days
Could you recommend alternatives?

Regards

**Comment by scriptkidde** [comment]
cheat issue

**Comment by TauCSGO** [comment]
> cheat issue

The cheat has no problems, it is the same cheat dll that currently works correctly with vanguard loader

**Comment by scriptkidde** [comment]
yeah this loader has weird shit with its injection so its a cheat issue, eg osiris wont inject with this loader but sensum or ot3 would

**Comment by TauCSGO** [comment]
> yeah this loader has weird shit with its injection so its a cheat issue, eg osiris wont inject with this loader but sensum or ot3 would

Ah, thanks for the clarification
I just use osiris, because it is a cheat that they update all the time
sensum has constant updates to avoid detection?
They provide the source code?

thanks for the info
